internal/multitenancy/multitenancy.go

```diff
@@ -157,8 +157,9 @@ func (tdm *TenancyDatamodel) setupProject(ctx context.Context, project *nexus.Ru
 
 	// Label default template
 	var defaultTemplateName string
+	baselineRegex := regexp.MustCompile(`^baseline-v\d+\.\d+\.\d+`)
```

Comment:
move this  to global variable to `mustCompile` will not panic in runtime

we have this pattern across repo

Reply:
for example
https://github.com/open-edge-platform/cluster-manager/pull/69/files#diff-c2fed50258ac24013de3009cb221b1f602579553cb2e23efc0a50e35c8c8b8cdR20